lib/lita/handlers/jls.rb

```diff
@@ -137,6 +143,72 @@ def cla(msg)
         push_exception(e, :project => "#{user}/#{project}", :pr => pr)
       end
 
+      def migrate_pr(msg)
+        source_url = msg.match_data["source_url"]
+        destination_url = msg.match_data["destination_url"]
+        if source_url.nil? || destination_url.nil?
+          raise "Invalid paramaters provided #{msg}"
+        end
+
+        pr_num = source_url.strip.split("/").last
+        if pr_num.nil? || pr_num !~ /\d+/
+          raise "Invalid pull request provided #{source_url}"
+        end
```

Comment:
You don't need this validation if you use the `github_url_parser`, you can initialize the class with the `:link => :pull_request` option and the `validate!` method will validate the url correctly.